maxim/logger/livekit/stt.py

```
@@ -71,7 +69,7 @@ def handle_stt_audio_input(self, audio_frames):
         )
 
 
-def handle_stt_result(self, result, generation_id=None):
+def handle_stt_result(self: STT, result, generation_id=None):
```

Ruff (0.12.2)

`handle_stt_result` is too complex (11 > 10)

(C901)